back/app/policies/project_policy.rb

```diff
@@ -174,6 +187,18 @@ def active_moderator?
 
   private
 
+  def update_status?
+    admin_like? || record.ever_published? || record.review&.approved?
+  end
+
+  def admin_like?
```

Comment:
This is clear. I wonder a bit if we have such methods defined already and abstracted elsewhere? Feels like this is probably not the first time we need something like this so I thought we had a way of doing it already? Or is this a special case of some kind?

Reply:
I thought we could allow folder moderators to delete projects in their folder (currently, they cannot), but we didn't discuss it, so I left it as is.